docs/source/tagset/elemento-contrib-group.rst

```diff
@@ -1,21 +1,29 @@
 .. _elemento-contrib-group:
- 
+
 <contrib-group>
 ---------------
 
-Aparece em
+Aparece em:
+
   :ref:`elemento-article-meta`
- 
-Ocorre
+
+Ocorre:
+
   Uma vez
 
+Contém o grupo de elementos relativos à contribuição na elaboração do artigo. Os contribuintes mais frequentes são os autores pessoais, instituições e grupos de pesquisa.
+
+O elemento pode ou não conter a afiliação, sendo obrigatória quando o atributo tipo de contribuição (@contrib-type) for ``author``, sejam estes autores institucionais ou não.
+
+Os principais elementos de ``<contrib-group>`` são:
+
+* :ref:`elemento-contrib`;
+* :ref:`elemento-xref`;
+* :ref:`elemento-collab`;
+* :ref:`elemento-aff`; e
+* :ref:`elemento-role`.
+
+Exemplo:
```

Comment:
Sim. Não vejo problemas.

-Gustavo

On 30 de junho de 2016 at 11:47:11, Letícia Aquino (notifications@github.com) wrote:
> > Uma vez
> >
> > +Contém o grupo de elementos relativos à contribuição na elaboração do artigo. Os contribuintes  
> mais frequentes são os autores pessoais, instituições e grupos de pesquisa.
> > +
> > +O elemento pode ou não conter a afiliação, sendo obrigatória quando o atributo tipo  
> de contribuição (@contrib-type) for ``author``, sejam estes autores institucionais  
> ou não.
> > +
> > +Os principais elementos de ```` são:
> > +
> > +* :ref:`elemento-contrib`;
> > +* :ref:`elemento-xref`;
> > +* :ref:`elemento-collab`;
> > +* :ref:`elemento-aff`; e
> > +* :ref:`elemento-role`.
> > +
> > +Exemplo:
>  
> @gustavofonseca Podemos usar o mesmo?
>  
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/scieloorg/scielo_publishing_schema/pull/245/files/621028e5fa93deec14b2c39456543e0207637e59#r69145938